src/generator/backend/_scss/application.scss

```diff
@@ -149,61 +150,76 @@ body {
   margin-top: -5px;
 }
 
-.video-link > .fa {
-	font-size: 22px;
-	color: #757575;
-}
+.video-link {
 
-.video-link > .fa:hover {
-	color: #e52d27;
-}
+  margin-left: 3px;
+   > .fa {
+    font-size: 22px;
+    color: #757575;
+    &:hover {
+      color: #e52d27;
+    }
+  }
 
-.video-link  {
-	margin-left: 3px;
 }
 
-.session-link > .fa {
-	font-size: 18px;
-	color: #757575;
-}
+.session-link {
 
-.session-link > .fa:hover {
-	color: #2196F3;
-}
+  > .fa {
+    font-size: 18px;
+    color: #757575;
+
+      &:hover {
+        color: #2196F3;
```

Comment:
@championswimmer  , I think the same , doing that . But what should be the solution for values like $code-color . The $code-color is set as white in config , but overwritten in light and dark themes . If I replace all hardcoded white everywhere with $code-color , it will change to gray in all the places in light theme which we don't require .